sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCache.java

```diff
@@ -259,6 +260,8 @@ public Mono<Utils.ValueHolder<AddressInformation[]>> tryGetAddresses(RxDocumentS
             this.suboptimalServerPartitionTimestamps.remove(partitionKeyRangeIdentity);
         }
 
+        String callIdentifier = UUID.randomUUID().toString();
```

Comment:
are we trying to use the callerIdentifier here to differentiate different workflow? then should we use fixed value instead?

Reply:
Resolved offline